src/components/AdvancedOptions.tsx

```diff
@@ -130,7 +135,7 @@ export default function AdvancedOptions(): ReactElement {
           </div>
         </div>
         <div className={styles.parameter}>
-          <div className={styles.inputGroup}>
+          <div data-testid="inputGroup" className={styles.inputGroup}>
```

Comment:
Same thing as above. The naming for this would be better as something like `deadlineInputGroup`.